### ISSUE TITLE
some source cannot be downloaded from fr.sourceforge...

### ISSUE THREAD

**Issue by docxu233333333333333333** [fw4spl-org/fw4spl-doc#1]
when I tried to compile fw4spl-deps, I find that question.
I thought that using  sourceforge's mirror  (sourceforge.mirrorservice.org)   install of fr.sourceforge is a nice choice for other persons outside France .But some thing failed again(i am totally a novice about cmake :(
Can you help me solve it?

**Comment by fbridault** [comment]
Yeah indeed you're right, we should not use french mirrors by default. I'll try to fix this asap.
Otherwise, could you please report the issue on the appropriate repo tracker : https://github.com/fw4spl-org/fw4spl-deps ?
Thanks for your feedback.

**Comment by fbridault** [comment]
Ok I fixed that on my fork (see the link above) and now I'm waiting the pull-request to be approved. You can test it in between if you feel like.
